HoneyBadgers.ConsoleApp/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mail;
+using HoneyBadgers.Logic;
+using HoneyBadgers.Logic.Helpers;
+
+namespace HoneyBadgers.ConsoleApp.Repositories
+{
+    public class UserRepository : IUserRepository
+    {
+        public List<User> Users { get; private set; } = new List<User>();
+
+        public UserRepository()
+        {
+            Users.AddRange(FileDataReader.LoadUsers());
+        }
+
+        public void AddUser()
+        {
+            Console.WriteLine("Adding new user");
+            var user = GetUserData();
+            Users.Add(user);
+            Console.WriteLine("The user has been added correctly");
+        }
+
+        public User GetUserData()
```

Comment:
`GetUserData` 'brzmi' trochę jak weź dane użytkownika/jakiegoś użytownika. GetDataFromUser albo coś takiego może? InputUserData? Nie wiem :octocat:

Reply:
to może SetNewUser? Set by oznaczało że wprowadzamy ajkiegoś dane i też będzie jasne że jest to nowy użytkownik

Reply:
Brzmi dobrze :)